src/components/pages/ImageClassification/ImageAnnotationModal/usePointsGeoJson.js

```diff
@@ -0,0 +1,52 @@
+export const usePointsGeoJson = ({ dataToReview, map, imageScale }) => {
+  const halfPatchSize = dataToReview ? dataToReview.patch_size / 2 : undefined
```

Comment:
this one line is the only modified one. To make it a ternary so that the value can be undefined if `dataToReview` doesnt exist yet